frontend/src/pages/data-provider.tsx

```diff
@@ -21,10 +30,43 @@ export default function Home(): JSX.Element {
           <HowDatasetWorks />
           <AddDataSection />
           <HowToAddData />
-          <FaqsSection />
+          <FaqsSection faqs={faqs} />
           <NewsLetterSignUpSection />
         </div>
       </Layout>
     </>
   );
 }
+
+export const getServerSideProps: GetServerSideProps<any> = async () => {
+  const mddb = await clientPromise;
+
+  //fetch faqs
+  const faqsFiles = await mddb.getFiles({
```

Comment:
Deployment is broken https://tdc-data-portal-git-feature-homepage-functional-impl-datopian1.vercel.app/data-provider due to the `getServerSideProps` with `fs` and `mddb` calls in it

Reply:
fixed